### ISSUE TITLE
[TextField][Narrator] TextField error message is not announced by narrator appropriately onBlur of the input.

### ISSUE THREAD

**Issue by ketaniyengar-msft** [microsoft/fluentui#19706]
### Environment Information

- **Package version(s)**: Fluent 8
- **Browser and OS versions**: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)

### Describe the issue:

We have a TextInput element in the page which when exited (onBlur), a error message is set if the input value is incorrect based on some logic. When narrator is used to enter something inside the input and then Tab is pressed, narrator focusses on the next focusable element and then reads out the error message.

### Please provide a reproduction of the issue in a codepen:

This is the link to the codepen - https://codepen.io/ketaniyengar-msft/pen/OJgpVmv

Reproduction steps -
1. Start narrator and focus on the TextInput and then start typing something.
2. Press tab button and listen to narrator's announcement.

#### Actual behavior:

This shows the actual behavior that is happening -
![image](https://user-images.githubusercontent.com/87801215/132644005-9a779818-4b7a-42dd-9e94-8f630758cbbe.png)

#### Expected behavior:

In the above image the expected behavior is that the error message should be read immediately before the next focusable element gets attention. (In simple terms, **2** should precede **1**)

**Comment by gouttierre** [comment]
@ketaniyengar-msft - Thanks for filing this issue with us. Our team will have a look but to set expectations, we’re likely not going to get to it immediately due to the fact we’re currently focused on larger coherence work items.

@smhigley , @bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by khmakoto** [comment]
Hi @ketaniyengar-msft, I just double-checked with @smhigley and there is no author control or external requirement about how screen readers decide to prioritize focus announcements vs live region announcements, with the moving focus usually coming in first. Given that and the fact that the correct message is eventually being announced I'll go ahead and resolve this as "Not an issue". If you have any other things we can help you with please open another issue. Thanks!
